components/steering_wheel/include/Utility.h

```diff
@@ -55,3 +66,118 @@ static inline uint16_t u32CountLeadingZeroes(uint32_t x) { uint16_t c = 0U; whil
             (bit) = false;              \
         }                               \
     } while (zero())
+
+
+// FLAGS
```

Comment:
@JoshLafleur FYI this is the API I was referring to that you could use for the IO stuff

Reply:
Regarding what we spoke about for the bitfield correct? This is the workaround to not being able to reference elements of a bitfield? To be frank I am a bit confued on the conversion from name to flag, the name would it be a define? or one of the elements of the bitfield? Would like to see more the workings of this

Reply:
Discussed offline